src/global/variables.css

```diff
@@ -0,0 +1,8 @@
+:root {
```

Comment:
I don't know what the answer to this is yet, but we should document the difference between these variables, what's in `formation`, and what's on `vets-website`. Not that there necessarily is a difference in values, but that these values live in potentially three different places.

Bonus points for dropping a comment in here with a link to where you got these values.